omod/src/main/java/org/openmrs/module/metadatasharing/web/extension/html/AdminList.java

```diff
@@ -1,76 +0,0 @@
-/**
- * The contents of this file are subject to the OpenMRS Public License
- * Version 1.0 (the "License"); you may not use this file except in
- * compliance with the License. You may obtain a copy of the License at
- * http://license.openmrs.org
- *
- * Software distributed under the License is distributed on an "AS IS"
```

Comment:
Why did you delete this file?

Reply:
This class used `org.openmrs.module.web.extension`, unavailable on `2.0.0`. Also, this class is not used in the code. That's why I removed it.
Should I revert this change and use the legacy UI module to provide `org.openmrs.module.web.extension`?

Reply:
@dkayiwa I think using the legacy UI module is better, so I updated the PR.

Reply:
Without this class, do the module's admin links still display on the admin page? https://qa-refapp.openmrs.org/openmrs/admin/index.htm

Reply:
It doesn't, I updated the PR.